src/components/Permissions/AppPermissions.jsx

```diff
@@ -93,6 +94,12 @@ const PermissionsApplication = ({ t }) => {
 
   const isKonnector = type => type === 'io.cozy.konnectors'
 
+  const isNotLastItem = name =>
+    name !==
+    sortPermissionsByName(matchingQueryResult)[
```

Comment:
let's extract this in one single variable 🙏

Reply:
I don't understand what you mean. It is a function that returns either `true` or `false`

Reply:
You call several times `sortPermissionsByName` with the same arg `matchingQueryResult`

You should call only once `sortPermissionsByName(matchingQueryResult)`. You can do something like:

```
const sortedPermissionsByName = sortPermissionsByName(matchingQueryResult)
const isNotLastItem = name => {
    return (
      name !==
      sortedPermissionsByName[
        sortedPermissionsByName.length - 1
      ].name
    )
  }
  ```

and also there is in the component: 
```
{sortPermissionsByName(matchingQueryResult).map(
```

that you can change to: 
```
sortedPermissionsByName.map(
```